src/options.cpp

```diff
@@ -1945,6 +1945,12 @@ bool &nl_while_leave_one_liners()
 }
 
 
+bool &nl_for_leave_one_liners()
```

Comment:
Hehe... yeah, this is a pain right now; sorry. I'm working toward landing #1852, which will make this all *much* simpler. Meanwhile, thanks for putting up with this :slightly_smiling_face:.